src/js/utils/language.js

```diff
@@ -45,7 +51,13 @@ export function getLanguage() {
     return language || navigator.language || navigator.browserLanguage || navigator.userLanguage || navigator.systemLanguage;
 }
 
-function extractLanguage(doc) {
-    const htmlTag = doc.querySelector('html');
-    return htmlTag ? htmlTag.getAttribute('lang') : null;
+export const translatedLanguageCodes = ['ar', 'da', 'de', 'es', 'fr', 'it', 'ja', 'nb', 'nl', 'pt', 'ro', 'sv', 'tr', 'zh'];
+
+export function isTranslationAvailable(language) {
+    return translatedLanguageCodes.indexOf(language.substring(0, 2).toLowerCase()) >= 0;
+}
+
+export function loadJsonTranslation(base, languageCode, oncomplete, onerror) {
```

Comment:
It'd be helpful to have a comment explaining that we don't load `en` JSON i.e. english is default